### DISCUSSION TITLE
Radius value is not respected when looking for routes based on coordinates

### DISCUSSION THREAD

**Discussion by code-vagabond** [transitland/transitland#432] [Q&A]
When I use the Route REST API to call for all routes in a certain radius e.g 500 meters, the result would contain routes where the nearest stops radius-wise to the coordinate is over 1000 meters away. Is there anyway to deal with this ? Thanks in advance 

My request:
`https://api.transit.land/api/v2/rest/routes.geojson?include_geometry=true&include_alerts=false&lon=${coordinate.long}&lat=${coordinate.lat}&radius=500&limit=10&api_key=${apiKey}`

**Comment by irees** [comment]
hi @code-vagabond Hmm, I'm not seeing this behavior. Can you share the exact coordinates you are using? You can reply here or the email in my github profile.
